ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml

```diff
@@ -49,14 +49,20 @@ tests:
   secret:
     mount_path: /tmp/secret
     name: ols-codecov-token
-- as: e2e-ols-cluster
+- as: e2e-ols-cluster-4-15
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
```

Comment:
@syedriko forgot about this. For 4.15, it's `us-east-1`

Reply:
@JoaoFula I don't think there's a 4.15 cluster pool like that:

```
   * could not run steps: step [release:latest-e2e-ols-cluster-4-15] failed: failed to find a cluster pool providing the cluster: map[architecture:amd64 cloud:aws owner:ols product:ocp region:us-east-1 version:4.15] 
```

If I search https://docs.ci.openshift.org/docs/how-tos/cluster-claim/ for `ols`, there's only one cluster pool with `ols` for owner:
```
openshift-observability-cluster-pool	ols-ocp-4-16-amd64-aws-us-east-1	0	0	1	3	ocp-release-4.16.1-x86-64-for-4.16.0-0-to-4.17.0-0	ols
```